cloud.foundry.cli/src/test/java/cloud/foundry/cli/system/GetAllCommandTest.java

```diff
@@ -0,0 +1,145 @@
+package cloud.foundry.cli.system;
+
+import cloud.foundry.cli.crosscutting.mapping.YamlMapper;
+import cloud.foundry.cli.crosscutting.mapping.beans.*;
+import cloud.foundry.cli.system.util.ArgumentsBuilder;
+import cloud.foundry.cli.system.util.RunResult;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link cloud.foundry.cli.services.GetController}, specifically the default get command which fetches all
+ * the information and outputs them in the same format we expect for the input.
+ */
+public class GetAllCommandTest extends SystemTestBase {
+
+    @BeforeEach
+    private void cleanUpSpaceBefore() {
+        spaceConfigurator.clear();
+    }
+
+    @AfterEach
+    private void cleanUpSpaceAfterwards() {
+        spaceConfigurator.clear();
+    }
+
+    private void assertRootBeanIsValid(ConfigBean rootBean) {
+        assert rootBean.getApiVersion().startsWith("2.");
+
+        TargetBean target = rootBean.getTarget();
+        assert target.getEndpoint().equals(getCfApiEndpointValue());
+        assert target.getOrg().equals(getCfOrganizationValue());
+        assert target.getSpace().equals(getCfSpaceValue());
+
+        SpecBean spec = rootBean.getSpec();
+        // TODO: check if we can put in an absolute value once we use "throw-away spaces" created per test run
+        assert spec.getSpaceDevelopers().size() > 0;
+    }
+
+    private ConfigBean loadConfigBeanFromStdoutAndAssertItsValidity(String stdoutContent) {
+        ConfigBean rootBean = YamlMapper.loadBeanFromString(stdoutContent, ConfigBean.class);
+        assertRootBeanIsValid(rootBean);
+        return rootBean;
+    }
+
+    @Test
+    public void testEmptySpace() {
+        ArgumentsBuilder args = new ArgumentsBuilder()
+                .addArgument("get");
+
+        RunResult runResult = runBaseControllerWithCredentialsFromEnvironment(args);
+
+        String outContent = runResult.getStreamContents().getStdoutContent();
+        assert outContent.length() > 0;
+
+        // parse YAML from stdout and check it for validity
+        ConfigBean rootBean = loadConfigBeanFromStdoutAndAssertItsValidity(outContent);
+
+        SpecBean spec = rootBean.getSpec();
+        assert spec.getApps() == null;
+        assert spec.getServices() == null;
```

Comment:
A comment explaining, why the space developers are not checked here, would be good